squad/frontend/templates/404.html

```diff
@@ -0,0 +1,6 @@
+{% extends "squad/base.html" %}
+{% load squad %}
+
+{% block content %}
+<h1>Page not found!<h1>
+{% endblock %}
```

Comment:
It's used in this line:
testjob = get_object_or_404(TestJob, pk=testjob_id)
In general this template will be used for any 404 that might happen in squad. Do we have any other 404 template I'm not aware of?